code/game/machinery/seed_extractor.dm

```diff
@@ -28,6 +29,29 @@
 
 obj/machinery/seed_extractor/attackby(var/obj/item/O as obj, var/mob/user as mob)
 
+	// Emptying a plant bag
+	if (istype(O,/obj/item/weapon/storage/bag/plants))
+		var/obj/item/weapon/storage/P = O
+		var/loaded = 0
+		for(var/obj/item/seeds/G in P.contents)
+			if(contents.len >= 999)

```

Comment:
Moved the global to a define in setup.dm, for whatever god-forsaken reason it was being declared a second time in microwave.dm, which shouldn't even have been using it.